samples/teams-blazor-server/Pages/Index.razor

```diff
@@ -0,0 +1,30 @@
+@page "/"
+
+
+
+<h2>Hello, MGT world!</h2>
+
+<p>
+    Welcome to your new app. It uses the Microsoft Graph Toolkit to log you in and shows details for your user.
+    <br />
+    Click on the sign in button to login. The very first time the pop up will ask you for consent for your Azure AD app.
+</p>
+
+<div>
+    <mgt-teams-provider 
+        client-id="%YOUR_CLIENT_ID%"
+        auth-popup-url="%YOUR_AUTHPAGE%"
```

Comment:
The teams provider supports relative paths, so something like `/auth` should work while not asking the developer to put the full path.